Update steps and images for new project setup page (#9577)
circleci/circleci-docs: docs/guides/modules/about-circleci/pages/introduction-to-the-circleci-web-app.adoc

**Groups**: Provides a list of user groups that have been granted project-specific permissions. For more information, see the xref:permissions-authentication:manage-groups.adoc[Manage groups] page.

> **Project Setup**: Provides a list of pipelines that have been set up for the project and their associated triggers. See xref:orchestrate:pipelines.adoc[Pipelines Overview and Setup] for more information. This page includes the option to begin setting up pipelines if none exist. The setup UI for scheduled pipelines is located here (if available for your integration). To set up a xref:orchestrate:scheduled-pipelines.adoc[scheduled pipeline] you will define a timetable, parameter, and attribution to automatically run a pipeline when the criteria is met. Some integration types will see two pages in the sidebar for this section: *Pipelines* and *Triggers*.

Reviewer: ```suggestion
**Project Setup**: Provides a list of GitHub pipelines (integrated both either with the OAuth integration or the GitHub App integration) that have been set up for the project, and their associated triggers. See xref:orchestrate:pipelines.adoc[Pipelines Overview and Setup] for more information. This page includes the option to begin setting up pipelines if none exist. The setup UI for scheduled pipelines is located here (if available for your integration). To set up a xref:orchestrate:scheduled-pipelines.adoc[scheduled pipeline] you will define a timetable, parameter, and attribution to automatically run a pipeline when the criteria is met. Some integration types will see two pages in the sidebar for this section: *Pipelines* and *Triggers*.
```

**Groups**: Provides a list of user groups that have been granted project-specific permissions. For more information, see the xref:permissions-authentication:manage-groups.adoc[Manage groups] page.

> **Project Setup**: Provides a list of pipelines that have been set up for the project and their associated triggers. See xref:orchestrate:pipelines.adoc[Pipelines Overview and Setup] for more information. This page includes the option to begin setting up pipelines if none exist. The setup UI for scheduled pipelines is located either here or in the *Triggers* section (if available for your integration). To set up a xref:orchestrate:scheduled-pipelines.adoc[scheduled pipeline] you will define a timetable, parameter, and attribution to automatically run a pipeline when the criteria is met.

Reviewer: I think we should specify which integrations are shown in "Project setup" and which in "Pipelines & triggers"
```suggestion
**Project Setup**: Provides a list of GitHub pipelines that have been set up for the project and their associated triggers. See xref:orchestrate:pipelines.adoc[Pipelines Overview and Setup] for more information. Pipelines integrated with VCS different from GitHub are shown in the **Pipelines** tab. This page includes the option to create a first pipeline if none exist. The setup UI for scheduled pipelines is located here for GitHub pipelines or in the *Triggers* section for Bitbucket pipeliens. To set up a xref:orchestrate:scheduled-pipelines.adoc[scheduled pipeline] you will define a timetable, parameter, and attribution to automatically run a pipeline when the criteria is met.
```

> *Pipelines* and *Triggers*: Some integration types will see two pages in the sidebar for setting up and managing pipelines and triggers. The options available in these pages are equivalent to those available in the *Project Setup* page.

Reviewer: Here too, I think I'd go for something like this, but happy for you to keep the existing one (or a version of ) if you think it's better
```suggestion
*Pipelines* and *Triggers*: These pages let you set up pipelines and triggers when using the Gitlab, Bitbucket Cloud and Bitbucket Data Center integrations. 
```
